db.go

```diff
@@ -92,15 +92,21 @@ func (db *Database) Create() error {
 	return nil
 }
 
-func (db *Database) GetView(docName string, viewName string) (error, []byte) {
+func (db *Database) GetView(docName string, viewName string, key string) (error, []byte) {
```

Comment:
It's not a huge deal, but I typically return errors as a second value in a tuple. This is what Go authors usually do.